crates/matrix-sdk-crypto/src/identities/user.rs

```diff
@@ -587,6 +624,29 @@ impl OtherUserIdentityData {
         *m = self.master_key.as_ref().clone()
     }
 
+    /// Remember that this identity used to be verified at some point.
+    pub(crate) fn mark_as_verified_once(&self) {
```

Comment:
`mark_as_verified`?

Reply:
If we call it `mark_as_verified` it would be confusing I think. This is not marking the identity as verified (like it can be trusted from this point to send key). It's saying that this identity is expected to be verified starting this point.

Not sure what would be a good name :/ 
I thought of `pin_verification`, but can be `should_be_verified`?

Reply:
To be consistent with the `previously_verified` suggestion, I'll rename it to `mark_as_previously_verified`